app/views/songs/search.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="search_songs">
+	<%= form_for :songs, url: {action: 'search'}, html: {method: 'get'} do |f| %>
+		<%= f.text_field :title %>
+		<%= f.submit "Search" %>
+	<% end %>
+</div>
+
+<% @artist_search_results.each do |song| %>
+	<div class='song-cards'>
+		<% unless song.artwork == '' %>
+			<img class="searchart" src="http://images.gs-cdn.net/static/albums/<%= song.artwork %>" />
```

Comment:
Getting the url for the song card should happen in a helper method or the model. Then  you won't need a conditional to render the image.